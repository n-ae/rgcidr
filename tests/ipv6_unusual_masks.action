2001:db8::/49
