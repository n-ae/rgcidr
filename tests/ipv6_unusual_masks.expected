2001:db8::
2001:db8::1
2001:db8::8000:0
2001:db8::7fff:ffff
2001:db8:0:0:8000::
2001:db8:0:0:7fff:ffff:ffff:ffff
2001:db8:0:1::
